index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, do not hesitate to contact me.
+    </p>
+  </div>
+  <div class="connect1"> 
+   <a href="#"><p>LET US CONNECT</p></a>
+  </div>
+  <div class=" social_links1">
+    <ul>
+      <li>
+        <a href="#">
+          <img class="twit1-icon" src="./images/twitter.png" alt="twit-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="git1-icon" src="./images/github.png" alt="git-icon">
+        </a>
+      </li>
+      
+      <li>
+        <a href="#">
+          <img class="ln1-icon" src="./images/Linkedin-icon.png" alt="ln-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="m1-icon" src="./images/medium.png" alt="m-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="a1-icon" src="./images/angellist.png" alt="a-icon">
+        </a>
+      </li>
+     </ul>
+    
+  </div>
+  <div class="action11">
+  <a><p>Get my Resume</p></a>
+  </div>
+</div>
+</div>
+<div></div>
+<div class="skill-list">
+  <div class="frame11">
+    <div class="language">
+      <span class="text">Languages</span>
+      <span class="down-angle"><i class="fa-solid fa-angle-down"></i></span>
+    </div>
+    <ul>
+      <li class="ls">
+        <div class="language-list">
+          <ul>
+            <li><a href="#"> <img src="./images/js.png" alt=""><span>javaScript</span> </a></li>
+            <li><a href="#"><img src="./images/html.png" alt=""><span>HTML</span></a></li>
+            <li><a href="#"><img src="./images/css.png" alt=""><span>CSS</span></a></li>
+          </ul>
+        </div>
+      </li>
+        <li class="my-work">
+          <div class="frameworks">
+            <span >Frameworks</span>
+            <span class="greater-than"><i class="fa-solid fa-angle-right"></i></span>
+          </div>
+        </li>
+        <li class="my-work1">
+          <div class="skills">
+            <span>skills</span>
+            <span class="greater-than"><i class="fa-solid fa-angle-right"></i></span>
+          </div>
+        </li>
+    </ul>
+  </div>
+</div>
+</section>
+  <section class="contact-form1">
+      <div>
+          <h2>Contact Me</h2>
+          <p>If you have an application you are interested in developing,
+               a feature that you need built or a project that needs coding. 
+               I’d love to help with it </p>
+      </div>
+
+      <form action="https://formspree.io/f/mzbokowp" method="POST">
+          <input id="1" name="Name" type="text" placeholder="Full Name" maxlength="30" required>
+          <input id="2" name="email" type="email"  placeholder="Email" required>
+          <textarea id="3" name="Message"  rows="10" maxlength="500" placeholder="write your message here" required></textarea>
+          <button type="submit">Get it Touch</button>
```

Comment:
- [ ] Kindly remove the black border for this button just as the previous reviewer requested